pkg/stores/proxy/proxy_store.go

```diff
@@ -219,6 +214,12 @@ func (s *Store) ByNames(apiOp *types.APIRequest, schema *types.APISchema, names
 	if err != nil {
 		return nil, nil, err
 	}
+	if apiOp.Namespace == "*" {
+		// This happens when you grant namespaced objects with "get" by name in a clusterrolebinding. We will treat
+		// this as an invalid situation instead of listing all objects in the cluster and filtering by name.
+		objs.Items = []unstructured.Unstructured{}
+		return objs, nil, nil
+	}
```

Comment:
Does moving this after `s.list(...)` cause us to now list all objects in the cluster?

Reply:
Yes, it will.

Reply:
Resolving, discussed offline, will move above.